tests/integration/components/flexberry-datetime-picker-test.js

```diff
@@ -32,3 +32,36 @@ test('it renders', function(assert) {
   //Component does not support template block usage.
   assert.equal(this.$().text().trim(), '');
 });
+
+test('Testing dynamicProperties of flexberry-base-component', function (assert) {
+  assert.expect(2);
+
+  let propertyNameValue = 'class';
+  let propertyValueClass = 'firstClass secondClass';
+  let propertyValueClassChange = 'firstClass secondClass thirdClass';
+  let dynamicPropertiesTemp = { class: propertyValueClass, propertyName: propertyNameValue };
+
+  this.set('dynamicPropertiesTemp', dynamicPropertiesTemp);
+
+  this.render(hbs`
+    {{#flexberry-textbox
+      dynamicProperties = dynamicPropertiesTemp
+    }}
+      template block text
+    {{/flexberry-textbox}}
+  `);
+
+  let $component = this.$().children();
+
+  assert.strictEqual($component.hasClass(propertyValueClass), true,
+    'Component\'s container has \'firstClass and secondClass\' css-class');
+
+  Ember.set(dynamicPropertiesTemp, 'class', propertyValueClassChange);
+
+  this.set('dynamicPropertiesTemp', dynamicPropertiesTemp);
+
+  $component = this.$().children();
+
+  assert.strictEqual($component.hasClass(propertyValueClassChange), true,
```

Comment:
You should check class existance separately (.hasClass('firstClass'), .hasClass('secondClass'), .hasClass('thirdClass')), but not all at once (.hasClass('firctClass secondClass thirdClass')).